package.json

```diff
@@ -43,7 +42,7 @@
     "pretest": "npm run lint",
     "tests-only": "tap --no-check-coverage tests/unit.test.js tests/**/test.js",
     "test": "npm run tests-only",
-    "posttest": "aud --production"
+    "posttest": "npx 'npm@>=10.2' audit --production"
```

Comment:
```suggestion
    "posttest": "aud --production"
```

Revert. `aud` is a dependency and I believe `npx` itself has now been deprecated in favor of `npm exec`.

Ain't broke. Let's not waste time fixing.

Reply:
aud is unmaintained.  It is listed as deprecated at https://www.npmjs.com/package/aud . repo has been archived at https://github.com/ljharb/aud

https://osv.dev/vulnerability/GHSA-pfrx-2q88-qq97 exists in the current dep tree because of `aud`.

Reply:
It's not "unmaintained" or vulnerable, however, npm 10.2 obviates aud (my package), because `npm audit` runs without a lockfile.

It's fine to ask it to be separate from this PR, but this is a good change.

Reply:
I'd like to focus here on something small and releaseable. We're overdue to bump some of the spdx packages I maintain. If we can bump arborist without dropping Node versions, great.